syntax="proto2";

package aap_protobuf.service.control.message;

message BatteryStatusNotification {
  required uint32 battery_level = 1;
  optional uint32 time_remaining_s = 2;
  optional bool critical_battery = 3;
}